xcodeproj/internal/input_files.bzl

```diff
@@ -775,7 +775,7 @@ def _from_resource_bundle(bundle):
         non_arc_srcs = EMPTY_LIST,
         pch = None,
         resource_bundle_dependencies = bundle.dependencies,
-        resources = depset(bundle.resources),
+        resources = depset([(resource, None) for resource in bundle.resources]) if bundle.resources else EMPTY_DEPSET,
```

Comment:
Should this be `None`?

Reply:
not sure if I follow the question. Do you mean just set `None` instead of `EMPTY_DEPSET`?

Reply:
I mean, I don't think it should use `None`. Either this should already be a list of tuples, or we should associate a label here. Feels like it should already be a list of tuples though.

Reply:
ah makes sense

Reply:
I just remembered the reason I had this was because of these lines  
https://github.com/MobileNativeFoundation/rules_xcodeproj/blob/main/xcodeproj/internal/resources.bzl#L110-L128

Right now, I am post processing `root_bundle.resources` and `root_bundle.folder_resources` after the attributes are set, so `bundle` inside the `_from_resource_bundle` method doesn't have `resource` of type List of Tuples, but rather List of Strings (as expected). 

I think it makes sense for us to actually change `bundle.resources` and `bundle.folder_resources` to have types List of tuples to match. What do you think of that?

Reply:
I currently don't have an opinion on where it should be calculated. It should just be the most efficient as possible, and it should correctly set the label.